source/atlas-cli-deploy-docker.txt

```diff
@@ -167,6 +167,124 @@ deployment with `Docker Compose <https://docs.docker.com/compose/>`__.
 
          mongosh mongodb://root:root@localhost:27778/?directConnection=true
 
+Persist Data Across Runs with Docker Compose
+--------------------------------------------
+
+You can persist data across multiple runs with Docker Compose.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Create a script to start and stop your local deployment.
+
+      a. Copy and paste the following script and name the file
+         ``entrypoint.sh``:
+
+         .. code-block:: sh
+
+            #!/bin/bash
+            stop_atlas() {
+              echo "Stopping Atlas deployment..."
+              atlas deployments stop
+            }
+            start_atlas() {
+              echo "Starting Atlas deployment..."
+              atlas deployments start
+            }
+           trap 'stop_atlas' SIGTERM SIGINT
+           deployment_status=$(atlas deployments list | grep 'LOCAL')
+           if [[ -z "$deployment_status" ]]; then
+             echo "No local deployment found. Setting up..."
+             atlas deployments setup dev --bindIpAll --username root --password $LOCALDEV_PASSWORD --type local --port 27017 --force
+           else
+             if [[ $deployment_status == *"STOPPED"* ]]; then
+               start_atlas
+             fi
+           fi
+           while true
+           do
+             tail -f /dev/null & wait ${!}
+           done
+
+      #. Run the following command to make the script executable only 
+         by you:
+
+         .. code-block:: sh
+
+            chmod u+x entrypoint.sh
+
+   .. step:: Update your ``docker-compose.yaml`` file.
+
+      a. Update the ``docker-compose.yaml`` file to mount the ``data`` 
+         directory as a volume.
```

Comment:
```suggestion
      a. Update the ``docker-compose.yaml`` file to mount the necessary data 
         directories as a volumes.
```

Apologies, I forgot to make this small tweak in the google doc when we adjusted the compose file.